src/shared/extension/hooks/use-dao-handles.ts

```diff
@@ -0,0 +1,13 @@
+import { useCommandQuery } from 'shared/messaging';
+
+import { GetDaoHandlesCommand } from '../commands';
+
+export const useDaoHandles = (application: string) => {
```

Comment:
I think this1 is mapping from twitter to username on given application, right?
If yes than we should probably move this1 to `host/twitter` and maybe we can try to apply more meaningfull name like `useHandleToUsernameMap`

Reply:
Also maybe it's worth to change api endpoint url to make it more obvious that it's specific for twitter CC @lennardevertz

Reply:
I can change it to `/dao-twitter-handles` for example. Is that better?

Reply:
updated :)